sklearn/linear_model/_logistic.py

```diff
@@ -1683,6 +1688,10 @@ def fit(self, X, y, sample_weight=None):
             Array of weights that are assigned to individual samples.
             If not provided, then each sample is given unit weight.
 
+        **fit_params : dict
+            Parameters to pass to the `fit` method of the underlying
+            classifier.
```

Comment:
the underlying classifier wouldn't actually accept anything here. These parameters are only used to be passed to scorer and splitter, right?